easybuild/toolchains/iimkl.py

```diff
@@ -32,13 +32,13 @@
 * Kenneth Hoste (Ghent University)
 * Bart Oldeman (McGill University, Calcul Quebec, Compute Canada)
 """
-from distutils.version import LooseVersion
 import re
 
 from easybuild.toolchains.iccifort import IccIfort
 from easybuild.toolchains.intel_compilers import IntelCompilersToolchain
 from easybuild.toolchains.fft.intelfftw import IntelFFTW
 from easybuild.toolchains.linalg.intelmkl import IntelMKL
+from easybuild.tools import LooseVersion
```

Comment:
This import of LooseVerstion differs from the one below in tools/__init__.py, why?

Reply:
Because in order to do `from easybuild.tools import LooseVersion` we need to import it in the tools `__init__.py`